### PR TITLE
Add Automatus Testing

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:
Adds couple of quick sanity tests for Automatus.
#### Rationale:

To help find Automatus issues.

### CODE REVIEW

File: products/fedora/profiles/test.profile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+title: 'Automatus Testing Profile'
+
+
+description: |-
+    This profile contains a few rules used to test Automatus in profile and combined modes.
+    This profile is testing only and should not be shipped.
+
+selections:
+    - package_tmux_installed
+    - file_groupowner_etc_passwd
```
Comment (jan-cerny) [inline products/fedora/profiles/test.profile:17]: missing OEL

**Comment by jan-cerny** [comment]
@Mab879 How can we verify that it works? It doesn't run in this PR.

**Comment by Mab879** [comment]
@jan-cerny I have made some updates to always run these tests.

**Review by jan-cerny — APPROVED** [review]
I have seen that there exists a job "Automatus Sanity / Run Tests" and that it has been executed and also I have seen its output.

**Pull request merged**